src/System.Management.Automation/engine/CoreAdapter.cs

```diff
@@ -4661,6 +4662,221 @@ protected override PSMemberInfoInternalCollection<T> GetMembers<T>(object obj)
     }
 
     /// <summary>
+    /// Deals with DataRow objects
```

Comment:
Never mind, I will revert this change, as I didn't add any tests for the DataRow/DataRowView adapters anyway.